XCUITests/FxScreenGraph.swift

```diff
@@ -187,16 +213,26 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
 
 extension Navigator {
     // Open a URL. Will use/re-use the first BrowserTab or NewTabScreen it comes to.
-    func openURL(urlString: String) {
+    func openURL(urlString: String, regularBrowsing: Bool = true) {
```

Comment:
As commented below, if regularBrowsing is false, we are browsing in private mode, and so we are redirected to the correct scenes, would that be ok according to your comment @jhugman?